components/progressbar/stories/meter.template.js

```diff
@@ -67,5 +67,27 @@ export const MeterGroup = (args, context) => html`
 				}, context)}
 			</div>
 		`)}
+
+		<!-- The gradient story below supports linear-gradients used by Express. For use cases that require a custom linear-gradient for any --mod-*-{fill} properties, set those custom properties in CSS. -->
```

Comment:
I wouldn't display this as an HTML comment, as it'll show in the source (including "show code" user viewable source in the docs).

Reply:
That is a good point. I wanted that comment to be closer to the gradient story itself, but I didn't think about the fact that html comments would show in the source code. Thanks for the keen eye! https://github.com/adobe/spectrum-css/pull/2929/commits/e3f078b98a6bd4c216ee6a0a8618ad778d744c61